README.md

```diff
@@ -1,6 +1,5 @@
-# Snappy Nextcloud
-
-Nextcloud server packaged as a snap. It consists of:
+# Nextcloud-snap, your Nextcloud in a Snap
```

Comment:
I don't think there's a reason to hyphenate this anywhere. Just call it the Nextcloud snap. This comment applies throughout this README and the wiki, please.

```suggestion
# Nextcloud in a snap
```